programs/perf-bench-numa-mem/run

```diff
@@ -12,10 +13,18 @@
 . $LKP_SRC/lib/reproduce-log.sh
 . $LKP_SRC/lib/debug.sh
 
+perf_path=""
+
+if [ -f /lkp/benchmarks/perf/perf ]; then
+	perf_path=/lkp/benchmarks/perf/perf
+else
+	perf_path=/usr/bin/perf
+fi
```

Comment:
thanks, for above lines, how about using `set_perf_path "/lkp/benchmarks/perf/perf"` like others? And the later call of perf can also changed to $perf.

Reply:
Thanks for your review, tried the same and it's working good. I have also pushed the changes.

Reply:
got it, and may need one more change that actually the whole if/else/fi can be replaced by one set_perf_path "/lkp/benchmarks/perf/perf", which will decide to use "/lkp/benchmarks/perf/perf" or /usr/bin/perf inside.

Reply:
Changed that and tested the same both the cases ("/lkp/benchmarks/perf/perf" and "/usr/bin/perf inside.") are working fine. Thanks.